integration_tests/e2e/prisonerChangeIncentiveLevelDetails.cy.ts

```diff
@@ -0,0 +1,122 @@
+import Page from '../pages/page'
+import PrisonerChangeIncentiveLevelDetailsPage from '../pages/prisonerIncentiveLevels/prisonerChangeIncentiveLevelDetailsPage'
+import PrisonerChangeIncentiveLevelConfirmationPage from '../pages/prisonerIncentiveLevels/prisonerChangeIncentiveLevelDetailsConfirmationPage'
+
+const IncentiveSummary = {
+  bookingId: -1,
+  iepDate: '2017-08-15',
+  iepTime: '2017-08-15T16:04:35',
+  iepLevel: 'Basic',
+  daysSinceReview: 1868,
+  nextReviewDate: '2022-09-26',
+  iepDetails: [
+    {
+      bookingId: -1,
+      iepDate: '2017-08-15',
+      iepTime: '2017-08-15T16:04:35',
+      agencyId: 'MDI',
+      iepLevel: 'Basic',
+      userId: 'INCENTIVES_API',
+      comments: 'INCENTIVES_API_COMMENT',
+    },
+  ],
+}
+context('Prisoner change incentive level details', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubFallbackHeaderAndFooter')
+    cy.task('stubNomisUserRolesGetCaseloads')
+    cy.task('stubManageUser')
+    cy.task('stubNomisUserRolesApiUserCaseloads')
+    cy.task('stubPrisonApiLocations')
+    cy.task('stubPrisonIncentiveLevels')
+
+    const roles = ['ROLE_MAINTAIN_IEP']
+    cy.task('stubSignIn', { roles })
+    cy.navigateToChangePrisonerIncentiveLevelDetails().then(result => {
```

Comment:
for ever `cy.navigateToChangePrisonerIncentiveLevelDetails()` call, you don’t need the `.then(…)` cos the "result" is not being used